doc/source/whatsnew/v0.23.0.txt

```diff
@@ -1099,6 +1099,7 @@ I/O
 - Bug in :meth:`pandas.io.json.json_normalize` where subrecords are not properly normalized if any subrecords values are NoneType (:issue:`20030`)
 - Bug in ``usecols`` parameter in :func:`pandas.io.read_csv` and :func:`pandas.io.read_table` where error is not raised correctly when passing a string. (:issue:`20529`)
 - Bug in :func:`HDFStore.keys` when reading a file with a softlink causes exception (:issue:`20523`)
+- Bug in :class:`HDFStore` for Series and empty DataFrames with timezone-aware data (:issue:`20594`)
```

Comment:
this is not very clear, is this for Series generally and empty DataFrames? this is *only* for a fixed store.